packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -336,5 +340,103 @@ tbody.spectrum-Table-body {
     --spectrum-table-col-resize-nubbin-icon-color: HighlightText;
     --spectrum-table-body-resize-indicator-background-color: Highlight;
     --spectrum-table-divider-border-color: CanvasText;
+
+    .spectrum-Table-cell,
+    .spectrum-Table-headCell,
+    .spectrum-Table-headCellButton {
+      --spectrum-table-cell-border-color-key-focus: Highlight;
```

Comment:
what is conflicting with this one to make the selectors necessary?
`--spectrum-table-cell-border-color-key-focus` is set under the .spectrum class, defining it under .spectrum-Table should be sufficient

if it's not, try changing the selector in this forced-colors: active media query from
`.spectrum-Table` -> `.spectrum-Table.spectrum-Table`